packages/arb-token-bridge-ui/synpress.cctp.config.ts

```diff
@@ -0,0 +1,163 @@
+
+if (!process.env.PRIVATE_KEY_CCTP) {
+  throw new Error('PRIVATE_KEY_CCTP variable missing.')
+}
+
+// Wallet funded on Sepolia and ArbSepolia with ETH and USDC
+const localWallet = new Wallet(process.env.PRIVATE_KEY_CCTP)
+// Generate a new wallet every time
+const userWallet = Wallet.createRandom()
+
+if (!process.env.PRIVATE_KEY_CCTP) {
+  throw new Error('PRIVATE_KEY_CCTP variable missing.')
+}
+
+if (!process.env.PRIVATE_KEY_USER) {
+  throw new Error('PRIVATE_KEY_USER variable missing.')
+}
+
+async function fundWallets() {
+  const userWalletAddress = await userWallet.getAddress()
+  console.log(`Funding wallet ${userWalletAddress}`)
+
+  /**
+   * We need 0.0002 USDC per test (0.0001 for same address and 0.0001 for custom address)
+   * And in the worst case, we run each tests 3 time
+   */
+  const usdcAmount = utils.parseUnits('0.0006', 6)
+  const ethAmountSepolia = utils.parseEther('0.01')
+  const ethAmountArbSepolia = utils.parseEther('0.002')
+  const ethPromises: (() => Promise<void>)[] = []
+  const usdcPromises: (() => Promise<void>)[] = []
+
+  if (tests.length === 2) {
+    ethPromises.push(() =>
+      fundEth({
+        address: userWalletAddress,
+        provider: sepoliaProvider,
+        sourceWallet: localWallet,
+        amount: ethAmountSepolia
+      })
+    )
+    ethPromises.push(() =>
+      fundEth({
+        address: userWalletAddress,
+        provider: arbSepoliaProvider,
+        sourceWallet: localWallet,
+        amount: ethAmountArbSepolia
+      })
+    )
+    usdcPromises.push(
+      () =>
+        fundUsdc({
+          address: userWalletAddress,
+          provider: sepoliaProvider,
+          networkType: 'parentChain',
+          sourceWallet: localWallet,
+          amount: usdcAmount
+        }),
+      () =>
+        fundUsdc({
+          address: userWalletAddress,
+          provider: arbSepoliaProvider,
+          networkType: 'childChain',
+          sourceWallet: localWallet,
+          amount: usdcAmount
+        })
+    )
+  } else if (tests[0].includes('deposit')) {
+    ethPromises.push(() =>
+      fundEth({
+        address: userWalletAddress,
+        provider: sepoliaProvider,
+        sourceWallet: localWallet,
+        amount: ethAmountSepolia
+      })
+    )
+    usdcPromises.push(() =>
+      fundUsdc({
+        address: userWalletAddress,
+        provider: sepoliaProvider,
+        networkType: 'parentChain',
+        sourceWallet: localWallet,
+        amount: usdcAmount
+      })
+    )
+  } else {
+    ethPromises.push(() =>
+      fundEth({
+        address: userWalletAddress,
+        provider: arbSepoliaProvider,
+        sourceWallet: localWallet,
+        amount: ethAmountArbSepolia
+      })
+    )
+    usdcPromises.push(() =>
+      fundUsdc({
+        address: userWalletAddress,
+        provider: arbSepoliaProvider,
+        networkType: 'childChain',
+        sourceWallet: localWallet,
+        amount: usdcAmount
+      })
+    )
```

Comment:
There are quite some cases which might be complex to debug later. Is there a way we can make this more efficient? Maybe have a fixed USDC and ETH amount being funded for any case and remove these cases? or if we decide to go ahead with this, maybe we should comment extensively.